Formula/p/pinentry-mac.rb

```diff
@@ -20,22 +20,25 @@ class PinentryMac < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
-  depends_on xcode: :build
-  depends_on "gettext"
-  depends_on "libassuan@2"
+  depends_on xcode: :build # for ibtool
+  depends_on "libassuan"
+  depends_on "libgpg-error"
   depends_on :macos
 
-  on_ventura :or_newer do
-    depends_on "texinfo" => :build
-  end
+  # Apply commit from `pinentry` to allow newer `libassuan`
+  # Ref: https://github.com/gpg/pinentry/commit/a39ba412ab24721d4edb6476156371f8bf1d3ff9
+  patch :DATA
```

Comment:
Can we send this patch to https://github.com/GPGTools/pinentry?

They'll probably adopt it eventually, but they don't have allow issues so a PR is probably the best way to reach them.